### ISSUE TITLE
Httpserver won't display any alerts, even if the Konami Code is entered correctly.

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/konami-code-lab#25]
Hello. There appears to be something wrong with the httpserver for this lab. Despite my code passing the tests, the browser will not display my alert if the Konami Code is entered correctly. In fact, even if an alert is written into the console itself, it will not display. I should note that when I create alerts in other tabs on my browser, the alert() function works just fine. 

No rush, but it would be nice to see the alert for typing in the Konami Code correctly. From what I hear, I am not the only student who has been having this issue, either.

Thanks for looking into this!

Steven

**Comment by mpothier** [comment]
Hi Steven,
I'm having the same issue here. I know that I'm hitting the alert because I can get a console.log() message to register on the line before it, but the alert itself doesn't register.

**Comment by junglerhino37** [comment]
Same

**Comment by RZacchia** [comment]
I have the exact same issue. I also tested with console.log() and it worked fine.

**Comment by kyoung90** [comment]
Same issue

**Comment by juan-nicolas** [comment]
Same...

**Comment by justbennett** [comment]
Issues #25-#28 seem to all be related. The tests that are running in the browser are somehow capturing the alert. It took me a while to figure out that the test also runs `init()` automatically. I was calling it in my js file and so every `keydown` event was triggering twice. I'm also unable to pass the "Run local Test" milestone on this lab. The local test is run in the browser and shows passing, but nothing happens on my curriculum page. I can however, pass the "Submitted Pull Request" milestone.

**Comment by s-blais** [comment]
Same same same. I was passing the tests but whyyyyy was I not seeing the alert? (the lack of payoff was disappointing)... I created and opened a local html file and ran my code from the js file just to prove to myself that I really did get it correct. Hey, there's the alert!

Then a good bit of frustration trying to achieve a green light on the test before submitting (despite the httpserver window giving me green checkmarks many times over). I gave up on getting a green light and submitted anyway. Then couldn't advance to the next lesson. After exiting out to the learn homepage and returning to the lesson, it was then marked as completed and the next lesson button now lets me progress...  whew
